Dockerfile

```diff
@@ -22,23 +22,32 @@ RUN ln -s /var/www/hw/ /var/www/hw/secure
 COPY /etc/hw-http*.conf /etc/apache2/sites-available/
 COPY /etc/hw.conf /etc/apache2/
 COPY /etc/apache2.conf /etc/apache2/apache2.conf.append
+COPY /scripts/httpd-foreground /usr/local/sbin/httpd-foreground
+RUN mkdir -p /etc/apache2/ssl
+COPY /etc/ssl/self.crt /etc/apache2/ssl/self.crt
+COPY /etc/ssl/self.key /etc/apache2/ssl/self.key
 RUN cat /etc/apache2/apache2.conf.append | tee -a /etc/apache2/apache2.conf
 RUN htpasswd -cb /etc/apache2/.htpasswd user user
 RUN a2dissite 000-default.conf
 RUN a2enmod ssl headers
+RUN a2disconf other-vhosts-access-log
 RUN sed -i 's/Listen 80/Listen 8080/g' /etc/apache2/ports.conf
 RUN sed -i 's/Listen 443/Listen 8443/g' /etc/apache2/ports.conf
 RUN a2ensite hw-http.conf hw-https.conf
 
 # System account
-RUN useradd -r -u 1001 user
-RUN chown -RL user: /etc/ssl/private/ /var/log/apache2/ /var/run/apache2/
+#RUN useradd -r -u 1001 user
```

Comment:
Hi,
Running `bash` in a running container shows this:
`I have no name!@ddff265e74fc:/$`
Could we not comment out this line?
Thanks.